route.go

```diff
@@ -81,7 +81,19 @@ func (r *AppRoute) ProxyHandle(method string, path string, proxy *httputil.Rever
 		ctx.Logger.Print("Completed ", ctx.Response.Status(), " ", http.StatusText(ctx.Response.Status()), " in ", time.Since(ctx.startedAt))
 
 		r.server.reuse(ctx)
-	})
+	}
+
+	if method == "Any" {
```

Comment:
change to `*` wildcard？ and preferred `switch` statements instead of `if ... else ...`